docs/crud/crudspec.md

```diff
@@ -15,8 +15,8 @@ For every model object in your schema we can provide the folowing operations:
 - UPDATE
```

Comment:
This got oudated. Instead of investing time here  I will prefer to start moving things to CRUDSpec instead and link it in our docs.

Reply:
Agreed, so did not bother touching it.